test/e2e/storage/testsuites/provisioning.go

```diff
@@ -866,15 +866,15 @@ func prepareSnapshotDataSourceForProvisioning(
 	}
 
 	// write namespace to the /mnt/test (= the volume).
-	tests := []e2evolume.Test{
```

Comment:
I think that by commenting this block you might have made your test pass but broken other tests

Reply:
I completely agree Mau. I'll continue to further test the different approaches we discussed over in slack.